test/contracts/workers/set_worker.js

```diff
@@ -105,6 +107,31 @@ module.exports.perform = (accounts) => {
 
   });
 
+
+  it('pass to set worker at lower deactivation height if worker was already set to a higher deactivation height', async () => {
+       
+    // remove existing workers
```

Comment:
this first cleanup is not needed;  ok to cleanup; but then a new worker contract is cleaner;  in the second paragraph you set the worker to a new height already

Reply:
Sure, removed the clean up